app/tab_info.py

```diff
@@ -2,16 +2,84 @@
 """Info tab."""
 import streamlit as st
 
+from app.ptxboa_functions import read_markdown_file
+
 
 def content_info():
-    with st.expander("What is this?"):
+    st.image("img/logos.png")
+
+    with st.container(border=True):
+        st.markdown(read_markdown_file("md/info_intro.md"))
+
+        st.image("img/costs_of_hydrogen.png", width=800)
+
+    with st.container(border=True):
         st.markdown(
-            """
-**Info**
+            "### What's in | What's out in the tool's calculations and assumptions?"
+        )
 
-Information on product details of the PTX Business Opportunity Analyser
- including a citation suggestion of the tool.
+        step = st.radio(
+            "Select process step:",
+            [
+                "General",
+                "Electricity generation",
+                "Electrolysis",
+                "Derivate production",
+                "Transport",
+            ],
+            horizontal=True,
+        )
+        st.image("img/inout_header.png", width=800)
+        st.image(f"img/inout_{step}.png", width=800)
```

Comment:
I would rename the files to have lowercase names with undersocores instead of spaces and load with the following line:

```suggestion
        st.image(f"img/inout_{step.replace(" ", "_").lower()}.png", width=800)
```